### PR TITLE
add circle tool tutorial

### PR DESCRIPTION
**Pull request by sasha-astiadi**
asked by kds to add circle tool tutorial https://github.com/threefoldtech/info_grid/issues/79

### CODE REVIEW

File: src/collaboration_tools/collaboration_tools.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<h1> ThreeFold's Collaboration Tools </h1>
+
+<h2>Table of Contents</h2>
+
+- [Circle Tool](../circle_tool.md)
```
Comment (Mik-TF) [inline src/collaboration_tools/collaboration_tools.md:5]: This link doesn't work.
Write this instead:

- [Circle Tool](./circle_tool.md)

(only one dot)

**Comment by Mik-TF** [comment]
General Comment:

* To see if the changes you made to the Threefold Manual are correct, download the branch locally:
  * `git clone -b development_circletools https://github.com/threefoldtech/info_grid`
* Go to the folder `info_grid`
* Run the mdbook (here port 3031)
  * `mdbook serve --port 3031`
* Check the TF Manual on your local browser
  * `http://localhost:3031/`

**Comment by Mik-TF** [comment]
The images do not work.

Same as with the link above, you need only one dot

Not working
``

Working
``

**Comment by Mik-TF** [comment]
@sasha-astiadi 
I did a commit with the changes needed so the pictures and links are working.

**Comment by Mik-TF** [comment]
Screen Shot 2023-05-05 at 12 44 18 PM

@sasha-astiadi 

I also updated the TOC so we can have a quick overview of the content.

**Comment by Mik-TF** [comment]
@xmonader 
@AhmedHanafy725 
@ashraffouda 
@mohamedamer453 

If the changes are good with you, I say this PR can be merged.

**Review by mohamedamer453 — APPROVED** [review]
lgtm.

I ran it locally and everything is working fine. the images are rendered correctly and all links are working and point to the correct sections. and the documentation itself is good and has no issues.

**Comment by mohamedamer453** [comment]
After approving i realized that there's a conflict in the `SUMMARY.md` since another pr was merged yesterday. so please fix the conflict first.

**Comment by sasha-astiadi** [comment]
@mohamedamer453 just resolved the conflict!

**Pull request merged**